### ISSUE TITLE
Replace libraries with libs from Orbit

### ISSUE THREAD

**Issue by RobertHilbrich** [RobertHilbrich/assist-public#105]
We should only have the libraries local, that are not part of the official Eclipse Orbit repository. 

This affects:
- apache commons math
- logback
- swtchart

We should work with the following p2 repository: http://download.eclipse.org/tools/orbit/downloads/drops/R20180330011457/repository

This needs to happen before we can add the AMALTHEA Model dependency, which requires Apache commons maths...

**Comment by RobertHilbrich** [comment]
Removed math3 and swtchart. Logback cannot be removed easily - requires a lot of fiddling and an extension to the plugin in the orbit to have the custom AssistConsoleAppender working

Logging can be conveniently configured programmatically: http://www.vogella.com/tutorials/EclipseLogging/article.html

But this would require the logback plugin to export the logback-classic.jar - which would lead to the warning about multiple logback versions in the classpath. So we will stick to the old way.
